_benchmark/reference/workloads/test-procedures.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: default
+title: test_procedures
+parent: Workload reference
+grand_parent: OpenSearch Benchmark Reference
+nav_order: 100
+---
+
+# test_procedures
+
+If your workload only defines one benchmarking scenario specify the schedule on top-level. Use the challenge element if you want to specify additional properties like a name or a description. You can think of a challenge as a benchmarking scenario. If you have multiple challenges, you can define an array of challenges.
+
+This section contains one or more challenges which describe the benchmark scenarios for this data set. A challenge can reference all operations that are defined in the operations section.
+
+Parameter | Required | Type | Description
+:--- | :--- | :--- | :---
+`name` | Yes | String | The name of the test procedure. When naming the test procedure, do not use spaces so that the name is easy to enter on the command line.
+`description` | No | String |  A human readable description of the test procedure.
+`user-info` | No | String | A message that is printed at the beginning of the test intended to notify the user about important information related to the test, such as deprecations.
+`default` | No | Boolean | When set to `true`, OpenSearch Benchmark selects this test procedure by default if the user did not specify another `test-procedure` on the command line. If your workload only defines one challenge, it is implicitly selected as default, otherwise you need to define `"default": true` on exactly one challenge.
```

Comment:
```suggestion
`default` | No | Boolean | When set to `true`, OpenSearch Benchmark selects this test procedure by default if the user did not specify another `test-procedure` on the command line. If your workload only defines one test procedure, it is implicitly selected as default, otherwise you need to define `"default": true` on exactly one challenge.
```